### PR TITLE
DOCSP-31899 Update docs for Creating an export bucket to s3

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
## Proposed changes

Update docs for Creating an export bucket to s3. Specifically, add link to Atlas docs where we describe how to create and retrieve Cloud Provider role access for Atlas.

_Jira ticket:_ DOCSP-31899

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

File: internal/usage/usage.go [hunk 1]
```diff
@@ -377,7 +377,7 @@ dbName and collection are required only for built-in roles.`
 	EnableRealtimePerformancePanel            = "Flag that enables the Real Time Performance Panel project setting."
 	DisableRealtimePerformancePanel           = "Flag that disables the Real Time Performance Panel project setting."
 	CloudProvider                             = "Name of the provider of the cloud service where Atlas can access the S3 bucket. Atlas only supports AWS."
-	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket."
+	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket. To learn more about setting up and retrieving cloud provider access role, see: https://www.mongodb.com/docs/atlas/security/set-up-unified-aws-access/."
```
Comment (sarahsimpers) [inline internal/usage/usage.go:380]:
```suggestion
	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket. To learn more about setting up and retrieving a cloud provider access role, see: https://www.mongodb.com/docs/atlas/security/set-up-unified-aws-access/."
```

**Review by sarahsimpers — DISMISSED** [review]
LGTM with one small nit. Also, you may want to use a dochub URL here instead of the docs URL, but that's a non-blocking suggestion.

**Comment by gssbzn** [comment]
@kanchana-mongodb you need to regenerate the docs

File: internal/usage/usage.go [hunk 2]
```diff
@@ -377,7 +377,7 @@ dbName and collection are required only for built-in roles.`
 	EnableRealtimePerformancePanel            = "Flag that enables the Real Time Performance Panel project setting."
 	DisableRealtimePerformancePanel           = "Flag that disables the Real Time Performance Panel project setting."
 	CloudProvider                             = "Name of the provider of the cloud service where Atlas can access the S3 bucket. Atlas only supports AWS."
-	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket."
+	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket. To learn more about setting up and retrieving a cloud provider access role, see: https://dochub.mongodb.org/core/set-up-unified-aws-access/."
```
Comment (sarahsimpers) [inline internal/usage/usage.go:380]: Need to remove the trailing slash, I tried navigating to the URL and it doesn't work with the trailing slash
```suggestion
	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket. To learn more about setting up and retrieving a cloud provider access role, see: https://dochub.mongodb.org/core/set-up-unified-aws-access."
```

**Review by sarahsimpers — CHANGES_REQUESTED** [review]

**Review by sarahsimpers — APPROVED** [review]
LGTM, thanks!

**Review by gssbzn — APPROVED** [review]

**Pull request merged**